lib/diff.go

```diff
@@ -26,7 +26,8 @@ type DiffStat = deepdiff.Stats
 // LeftSide set with the UseLeftPrevVersion flag.
 type DiffParams struct {
 	// File paths or reference to datasets
-	LeftSide, RightSide string
+	LeftSide  string `schema:"left_path",json:"left_path"`
+	RightSide string `schema:"right_path",json:"right_path"`
```

Comment:
Need spaces between the comma and "json" to make `go vet` happy. Also, sounds like we're aiming for camelCase instead of snake_case.